### PR TITLE
Addition of better comments for the Github Enterprise rule.

### PR DESCRIPTION
**Pull request by rtucker-mozilla**
@floatingatoll I've provided better documentation her per your review.
Thanks again for your time!

### CODE REVIEW

File: rules/GHE-Groups.js [hunk 1]
```diff
@@ -3,8 +3,15 @@ function (user, context, callback) {
     'EnEylt4OZW6i7yCWzZmCxyCxDRp6lOY0': 'mozilliansorg_ghe_saml-test-integrations_users',
   };
   
+  // We only care about SSO applications that exist in the applicationGroupMapping
+  // If the SSO ID is undefined in applicationGroupMapping, skip processing and return callback()
   if(applicationGroupMapping[context.clientID] !== undefined){
+    // If the SSO application is one this ruleset cares about, confirm that the
+    // user groups include the proper mapping from the hash
     if(!user.app_metadata.groups.includes(applicationGroupMapping[context.clientID])){
+        // End goal is to possibly have conditional endpoint pages on mana that we'll redirect to on and org by or bases.
```
Comment (floatingatoll) [inline rules/GHE-Groups.js:12]: This is good, but we do need to say something like below, before the details.

`// User is denied access this application, halt sign-in and redirect to help page.`

**Review by floatingatoll — CHANGES_REQUESTED** [review]

**Comment by floatingatoll** [comment]
I get the impression this PR has drifted out of sync. Rob, did we already incorporate this with other PRs?

**Comment by rtucker-mozilla** [comment]
The file is in it's correct and proper state

**Comment by floatingatoll** [comment]
Alright, wfm'ing this.

**Pull request closed without merging**